src/block-components/separator/style.scss

```diff
@@ -2,8 +2,9 @@
 
 .stk-block {
 	&:is(.stk-has-top-separator, .stk-has-bottom-separator) {
-		> *:not(.stk-video-background) { // Don't change the order of any video backgrounds, they stay at the back.
+		> *:not(.stk-video-background, .stk-separator) { // Don't change the order of any video backgrounds, they stay at the back.
 			z-index: 2;
+			position: relative;
 		}
 
 		.stk-separator ~ *:not(.stk-separator, .stk-video-background) {
```

Comment:
@bfintal when I only add a top separator this rule applies that's why I thought it was fine to add that